client/app/bundles/Restaurants/components/Filters/Checkbox.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class CheckBox extends React.Component {
+    render() {
```

Comment:
A nice convention we have in React is the deconstruction 
const { title, name,  toggle} = this.props;